BUTTON_IDEAS.md

```diff
@@ -15,35 +15,37 @@ When adding ideas, please add them in the right category (or add a new one if yo
 
 ## Animated
 
-|status  | Button name    | Description      | Submitted by |
-|--------|----------------|------------------|--------------|
-| &#9745;| ~~**Skew Button**~~| A button that skews on hover. | | 
-| &#9745;| ~~**Neon Button**~~| A button that glows on hover. |[Esakki Selvaraj](https://github.com/esakki-selvaraj) |
-| &#9744;| **Ripple Button**| A button with ripple effect on hover | [Maitryee-Choudhary](https://github.com/Maitryee-Choudhary)
+| status  | Button name           | Description                          | Submitted by                                                |
+| ------- | --------------------- | ------------------------------------ | ----------------------------------------------------------- |
+| &#9745; | ~~**Skew Button**~~   | A button that skews on hover.        |                                                             |
+| &#9745; | ~~**Neon Button**~~   | A button that glows on hover.        | [Esakki Selvaraj](https://github.com/esakki-selvaraj)       |
+| &#9744; | ~~**Ripple Button**~~ | A button with ripple effect on hover | [Maitryee-Choudhary](https://github.com/Maitryee-Choudhary) |
 
 ---
 
 ## Icon
 
-|status  | Button name    | Description      | Submitted by |
-|--------|----------------|------------------|--------------|
-| &#9745;| ~~**Plus button**~~| A button just with a plus icon | | 
-| &#9745;| ~~**Buy me a coffee button**~~|  | |
-| &#9744;| **Icon Button**| A button with icons enlargement and shrink effect on hover | [Souvik Guria](https://github.com/devSouvik)
+| status  | Button name                    | Description                                                | Submitted by                                 |
+| ------- | ------------------------------ | ---------------------------------------------------------- | -------------------------------------------- |
+| &#9745; | ~~**Plus button**~~            | A button just with a plus icon                             |                                              |
+| &#9745; | ~~**Buy me a coffee button**~~ |                                                            |                                              |
+| &#9744; | **Icon Button**                | A button with icons enlargement and shrink effect on hover | [Souvik Guria](https://github.com/devSouvik) |
```

Comment:
Change "&#9744;" to "&#9745;"

Reply:
It should be "&#9745" instead of "&#9744"

Reply:
@shahednasser I think it's fine now.